src/quo2/components/tags/tag.cljs

```diff
@@ -59,22 +59,36 @@
       label])])
 
 (defn tag
+  "opts
+   {:type         :icon/:emoji/:label
+    :size         32/24
+    :on-press     fn
+    :blurred      true/false 
+    :resource     icon/image
+    :labelled     true/false
+    :disabled     true/false}
+  
+   opts
+    - `blurred`  boolean: use to determine border color if the background is blurred
+    - `type`     can be icon or emoji with or without a tag label
+    - `labelled` boolean: is true if tag has label else false"
   [_ _]
   (fn [{:keys [id on-press disabled size resource active accessibility-label
                label type labelled blurred icon-color] :or   {size 32}}]
     (let [state (cond disabled :disabled active :active :else :default)
           {:keys [border-color blurred-border-color text-color]}
           (get-in themes [(theme/get-theme) state])]
-      [base-tag/base-tag {:id                  id
-                          :size                size
-                          :border-width        1
-                          :border-color        (if blurred
-                                                 blurred-border-color
-                                                 border-color)
-                          :on-press            on-press
-                          :accessibility-label accessibility-label
-                          :disabled            disabled
-                          :type                type
-                          :label               label}
-       [tag-resources size type resource icon-color label text-color labelled]])))
+      [rn/view {:align-items  :center}
```

Comment:
Add `:style` attribute please

Reply:
hey @ibrkhalil could just elaborate further to me why you recomend add the `:style` here

Reply:
Styles should go inside the style prop to differentiate them from normal props, I don't know if there's any drawbacks of not doing that but I'd say it's cleaner

Reply:
thanks @ibrkhalil